cirq/pasqal/pasqal_sampler_test.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for pasqal_sampler."""
+from os import getenv
+
+import numpy as np
+import sympy
+
+import cirq
+
+
+def _make_sampler() -> cirq.pasqal.PasqalSampler:
```

Comment:
Travis is run 10s of times a day.  Are we going to be hitting your device every time or is this a check to make sure that your API works? Anyway, I don't think this a job we should delegate to Cirq. We should mock this.